src/UI/TextField.elm

```diff
@@ -460,59 +491,99 @@ The result of this function is a ready-to-insert Elm UI's Element.
 
-        nonStatic content msg =
-            case content of
-                ContentSinglelineText ->
-                    inputAnyOptions cfg msg prop opt
-                        |> Input.text elAttrs
+        whenPassword { isVisible, isCurrent } =
+            if isCurrent then
+                inputPasswordOptions cfg msg prop opt isVisible
+                    |> Input.currentPassword elAttrs
 
-                ContentMultilineText ->
-                    inputMultilineOptions cfg msg prop opt
-                        |> Input.multiline elAttrs
+            else
+                inputPasswordOptions cfg msg prop opt isVisible
+                    |> Input.newPassword elAttrs
+    in
+    case prop.content of
+        ContentSinglelineText ->
+            inputAnyOptions cfg msg prop opt
+                |> Input.text elAttrs
 
-                ContentUsername ->
-                    inputAnyOptions cfg msg prop opt
-                        |> Input.username elAttrs
+        ContentMultilineText ->
+            inputMultilineOptions cfg msg prop opt
+                |> Input.multiline elAttrs
 
-                ContentPassword pswOpt ->
-                    whenPassword msg pswOpt
+        ContentUsername ->
+            inputAnyOptions cfg msg prop opt
+                |> Input.username elAttrs
 
-                ContentEmail ->
-                    inputAnyOptions cfg msg prop opt
-                        |> Input.email elAttrs
+        ContentPassword pswOpt ->
+            whenPassword pswOpt
 
-                ContentSearch ->
-                    inputAnyOptions cfg msg prop opt
-                        |> Input.search elAttrs
+        ContentEmail ->
+            inputAnyOptions cfg msg prop opt
+                |> Input.email elAttrs
 
-                ContentSpellChecked ->
-                    inputAnyOptions cfg msg prop opt
-                        |> Input.spellChecked elAttrs
+        ContentSearch ->
+            inputAnyOptions cfg msg prop opt
+                |> Input.search elAttrs
 
-        whenStatic value =
-            Text.subtitle2 value
-                |> Text.renderElement cfg
+        ContentSpellChecked ->
+            inputAnyOptions cfg msg prop opt
+                |> Input.spellChecked elAttrs
+
+
+staticView : RenderConfig -> Properties msg -> Options msg -> Element msg
+staticView renderConfig prop opt =
+    let
+        elAttrs =
+            attrs renderConfig prop opt
+
+        genericInputElement value =
+            value
+                |> Text.subtitle2
+                |> Text.renderElement renderConfig
                 |> Element.el elAttrs
 
-        whenPassword msg { isVisible, isCurrent } =
-            if isCurrent then
-                inputPasswordOptions cfg msg prop opt isVisible
-                    |> Input.currentPassword elAttrs
+        inputElement =
+            case prop.content of
+                ContentPassword _ ->
+                    "●"
```

Comment:
Why not asterisks? I know we used this character in the wms-web, but I don't think makes sense to change the format. I would suggest using asterisks.

Reply:
Because Firefox and WebKit-based browsers use filled-circles tas password security character